osu.Game/Audio/LoudnessNormalizationInfo.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using JetBrains.Annotations;
+using osu.Framework.Testing;
+using osu.Game.Database;
+using osu.Game.Utils;
+using Realms;
+
+namespace osu.Game.Audio
+{
+    [ExcludeFromDynamicCompile]
+    public class LoudnessNormalizationInfo : RealmObject, ILoudnessNormalizationInfo, IHasGuidPrimaryKey, IEquatable<LoudnessNormalizationInfo>, IDeepCloneable<LoudnessNormalizationInfo>
```

Comment:
Why is this necessary? Why is it a `RealmObject` not an `EmbeddedObject`? Why does it need to be a separate object in the first place?